test/util/branch-selector.html

```diff
@@ -21,68 +21,68 @@
-      assert.equal('test', branchSelector.selected);
-    });
+      test('take selecting branch works', function() {
+        Polymer.dom(branchSelector.root).querySelector('iron-selector').select('test');
+        assert.equal('test', branchSelector.selected);
+      });
 
-    test('selecting branch hide dropdown', function(done) {
-      MockInteractions.focus(branchSelector);
-      var ironSelector = Polymer.dom(branchSelector.root).querySelector('iron-selector');
-      ironSelector.select('test');
-      flush(function() {
-        MockInteractions.tap(ironSelector.selectedItem);
-        assert.isTrue(branchSelector.hideSelectionMenu);
-        done();
+      test('selecting branch hide dropdown', function(done) {
+        MockInteractions.focus(branchSelector);
+        var ironSelector = Polymer.dom(branchSelector.root).querySelector('iron-selector');
+        ironSelector.select('test');
+        flush(function() {
+          MockInteractions.tap(ironSelector.selectedItem);
+          assert.isTrue(branchSelector.hideSelectionMenu);
+          done();
+        });
       });
-    });
 
-    test('filtering branches sets value', function() {
-      var input = Polymer.dom(branchSelector.root).querySelector('paper-input')
-      input.value = 'ing';
-      assert.equal(input.value, branchSelector.filter);
-    });
+      test('filtering branches sets value', function() {
+        var input = Polymer.dom(branchSelector.root).querySelector('paper-input')
+        input.value = 'ing';
+        assert.equal(input.value, branchSelector.filter);
+      });
 
-    test('filtering branches hides items', function(done) {
-      var input = Polymer.dom(branchSelector.root).querySelector('paper-input')
-      input.value = 'ing';
-      var ironSelector = Polymer.dom(branchSelector.root).querySelector('iron-selector');
-      flush(function() {
-        assert.isTrue(ironSelector.items[0].hidden);
-        assert.isFalse(ironSelector.items[1].hidden);
-        done();
+      test('filtering branches hides items', function(done) {
+        var input = Polymer.dom(branchSelector.root).querySelector('paper-input')
+        input.value = 'ing';
+        var ironSelector = Polymer.dom(branchSelector.root).querySelector('iron-selector');
+        flush(function() {
+          assert.isTrue(ironSelector.items[0].hidden);
+          assert.isFalse(ironSelector.items[1].hidden);
+          done();
+        });
       });
-    });
 
-    test('tabbing can select item', function() {
-      var ironSelector = Polymer.dom(branchSelector.root).querySelector('iron-selector');
-      flush(function() {
-        MockInteractions.pressAndReleaseKeyOn(ironSelector, 9);
-        MockInteractions.pressAndReleaseKeyOn(ironSelector, 9);
-        MockInteractions.pressEnter(ironSelector);
-        assert.equal('test', branchSelector.selected);
+      test('tabbing can select item', function() {
```

Comment:
This test is async, so it needs `done` which must be called within the function in `flush`